examples/platform/linux/CommissionerMain.cpp

```diff
@@ -110,14 +113,50 @@ class MyCommissionerCallback : public CommissionerCallback
     }
 };
 
+AutoCommissioner gAutoCommissioner;
+
+class MyCredsIssuer : public ExampleOperationalCredentialsIssuer
```

Comment:
`MyCreds` is not a good name. Please use a better more descriptive name.

Reply:
updated name